packages/os/sundog.service

```diff
@@ -1,8 +1,8 @@
 [Unit]
 Description=User-specified setting generators
-# Need network access to support commands talking to IMDS.
-After=network-online.target apiserver.service early-boot-config.service
-Requires=network-online.target apiserver.service
+# Needs network access to support commands talking to IMDS.
+After=network-online.target early-boot-config.service
+Requires=network-online.target early-boot-config.service
```

Comment:
`early-boot-config` requires `apiserver`, so by just specifying `early-boot-config` we get the exact same behavior as specifying all the removed units.

Reply:
I'd prefer units to more fully specify their dependency chain, at least within a stage and for our own programs, rather than relying on transitive dependencies. That makes it easier to grep for dependencies in the source tree, and easier to reason about the impact of moving or reordering any particular service.